libs/fixture-generators/src/generate-election/config.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable vx/gts-jsdoc */
+import z from 'zod';
+
+const positiveInt = z.number().int().min(1);
+
+export const GenerateElectionConfigSchema = z.object({
+  numDistricts: positiveInt,
+  numPrecincts: positiveInt,
+  numBallotStyles: positiveInt,
+  numParties: positiveInt,
+  numContests: positiveInt,
+  numCandidatesPerContest: positiveInt,
+  maxContestVoteFor: positiveInt,
+  maxStringLengths: z.object({
+    title: positiveInt,
+    countyName: positiveInt,
+    stateName: positiveInt,
+    districtName: positiveInt,
+    precinctName: positiveInt,
+    partyShortName: positiveInt,
+    partyFullName: positiveInt,
+    partyAbbreviation: positiveInt,
+    contestTitle: positiveInt,
+    contestTermDescription: positiveInt,
+    candidateName: positiveInt,
+    contestBallotMeasureText: positiveInt,
+    contestBallotMeasureOptionLabel: positiveInt,
+  }),
+});
+
+export type GenerateElectionConfig = z.infer<
+  typeof GenerateElectionConfigSchema
+>;
+
+/**
+ * TODO once we set our system limits, we should set this default config to
```

Comment:
This comment makes sense to me and makes me wonder if we should also then have a basic text documentation for it autogenerated from that code when we get there.

Reply:
That could be cool!